Gruntfile.js

```diff
@@ -575,6 +584,29 @@ module.exports = function (grunt) {
         ]);
     });
 
+    grunt.registerMultiTask('swPrecache', function() {
+        var done = this.async();
+        var rootDir = this.data.rootDir;
+        var files =  (function() {
```

Comment:
Does this rely on globbing patterns? Can't we just do something like `[ "dist/**/*" ]`?